Makefile

```diff
@@ -192,6 +192,9 @@ test-internet: all
 test-debugger: all
 	$(PYTHON) tools/test.py debugger
 
+test-bugs: all
```

Comment:
To me, `broken` makes it sound like a place where we put tests that are broken, rather than a place where we have good tests that are failing because of bugs in the main code base. I prefer `test-bugs` or even `test-known-issues`.